pages/navigationMenu/index.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/**
+ * @defgroup pages_navigationMenu NavigationMenu Pages
+ */
+
+/**
+ * @file lib/pkp/pages/navigationMenu/index.php
+ *
+ * Copyright (c) 2014-2017 Simon Fraser University
+ * Copyright (c) 2003-2017 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @ingroup pages_navigationMenu
+ * @brief Handle requests for NavigationMenus functions.
+ *
+ */
+
+switch ($op) {
+	case 'index':
+	case 'view':
+	case 'preview':
+		define('HANDLER_CLASS', 'NavigationMenuItemHandler');
+		import('lib.pkp.pages.navigationMenu.NavigationMenuItemHandler');
+		break;
+}
+
+?>
```

Comment:
Right now, custom pages have a URL like `/navigationMenu/view/test`. I realize we probably need to have a base slug in there of some kind, but it probably shouldn't be `navigationMenu`, as the nav menu is meaningless to the content of the page.

Ideally, I'd like to see `/page/test`, getting rid of the `view` op altogether if possible. URL slugs can have a big impact on SEO, so keeping them clean and focused on the path is best. @asmecher may have some better ideas here.